packages/cli/src/lib/yargs-cli.integration.test.ts

```diff
@@ -118,4 +120,24 @@ describe('yargsCli', () => {
       }),
     );
   });
+
+  it('should parse compare options', async () => {
+    const parsedArgv = await yargsCli<GeneralCliOptions & CompareOptions>(
+      ['--before=source-report.json', '--after', 'target-report.json'],
+      {
+        options: { ...options, ...yargsCompareOptionsDefinition() },
```

Comment:
Do we need all other options here?

Reply:
All the other integration tests in this file include them, so I wanted to be consistent. And the `compare` command other common options as well, so including them makes it closer to the actual scenario.